build-tools/packages/build-cli/src/commands/check/policy.ts

```diff
@@ -0,0 +1,248 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as fs from "fs";
+import { EOL as newline } from "os";
+// eslint-disable-next-line camelcase
+import * as child_process from "child_process";
+// eslint-disable-next-line unicorn/import-style
+import * as path from "path";
+import { Flags } from "@oclif/core";
+import {
+    copyrightFileHeaderHandlers,
+    npmPackageContentsHandlers,
```

Comment:
import names should be fixed here: assertShortCode, copyrightFileHeader, dockerfilePackages, fluidCase, lockfiles, npmPackages, Handler